internal/service/searchindex/resource_search_index_test.go

```diff
@@ -177,6 +177,74 @@ func basicTestCase(tb testing.TB) *resource.TestCase {
 	}
 }
 
+func TestAccSearchIndex_withStoredSourceFalse(t *testing.T) {
```

Comment:
A lot of acceptance tests.
Can some of them be unit tests instead?

Reply:
in this case I don't think so because these are checking that Atlas is behaving as expected, they're not about checking our code